public/index.html

```diff
@@ -3,9 +3,10 @@
     <head>
         <meta charset="utf-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+        <link href="https://fonts.googleapis.com/css?family=Quicksand:300,400,500,700" rel="stylesheet" />
         <title>BNTI</title>
     </head>
-    <body>
+    <body style="margin:0">
```

Comment:
lets use this for resetting https://github.com/zacanger/styled-reset

Reply:
actually go with https://polished.js.org/docs/#normalize

Reply:
ref: https://stackoverflow.com/questions/6887336/what-is-the-difference-between-normalize-css-and-reset-css